server/rest/src/main/java/org/infinispan/rest/resources/LoginResource.java

```diff
@@ -48,7 +48,7 @@ public Invocations getInvocations() {
 
    private CompletionStage<RestResponse> login(RestRequest restRequest) {
       NettyRestResponse.Builder responseBuilder = new NettyRestResponse.Builder();
-      responseBuilder.status(HttpResponseStatus.TEMPORARY_REDIRECT).header("Location", accessGrantedPath);
+      responseBuilder.status(HttpResponseStatus.ACCEPTED).header("Location", accessGrantedPath);
```

Comment:
Is the location header still needed if you're not doing a redirect?

Actually, since the login action doesn't seem to be doing anything, do we still need a login action?

Reply:
Yes, it validates that the user can log in

Reply:
For the console is useful

Reply:
I mean having a particular endpoint for that

Reply:
so, sorry, I did not understand the question correctly. this is not used as I do whatever I want in the console. console redirect is in the front

Reply:
I was answering more to the login action need than the location header. location header can indeed be removed

Reply:
so we can also remove the whole /rest/v2/login, since it does nothing with this PR changes, right?

Reply:
I would like to have an endpoint that brings no data to authenticate that a user can actually connect. the config mech is non auth

Reply:
Can't you use ```http://localhost:11222/rest/v2/server``` it returns  ```401 Unauthorized``` if you try to call it without being authenticated or otherwise ```200``` with:
```json
{
  "version": "Infinispan 'Taedonggang' 12.1.0-SNAPSHOT"
}
```

Reply:
.... ok ....

Reply:
do we delete the endpoint then ?

Reply:
I don't think anyone else is using it apart from the console, but we could add a ```deprecated()``` in the invocation for now and remove on the next major

Reply:
:+1: to deprecate only, it's good to get into the habit of always not deleting things outright even when we don't know of any users.